llvm/lib/Target/AMDGPU/SIInsertWaitcnts.cpp

```diff
@@ -310,7 +310,14 @@ class WaitcntBrackets {
   bool counterOutOfOrder(InstCounterType T) const;
   void simplifyWaitcnt(AMDGPU::Waitcnt &Wait) const;
   void simplifyWaitcnt(InstCounterType T, unsigned &Count) const;
-  void determineWait(InstCounterType T, int RegNo, AMDGPU::Waitcnt &Wait) const;
+
+  void determineWait(InstCounterType T, RegInterval Interval,
+                     AMDGPU::Waitcnt &Wait) const;
+  void determineWait(InstCounterType T, int RegNo,
```

Comment:
Is "int RegNo" supposed to be an MCPhysReg?

Reply:
RegNo is the index to the register scoreboard inside WaitcntBrackets structure, not MCPhysReg